mapmssql2008.c

```diff
@@ -690,10 +690,10 @@ static char *strstrIgnoreCase(const char *haystack, const char *needle)
 {
   char    *hay_lower;
   char    *needle_lower;
-  int     len_hay,len_need;
+  size_t  len_hay,len_need, match;
+  int     found = -1;
```

Comment:
```
  bool found = false;
```

Reply:
The bool type isn't available (at least not for me using MSVC). I've updated to use MS_FALSE and MS_TRUE for clarity.